pykafka/producer.py

```diff
@@ -66,8 +66,10 @@ def __init__(self,
                  min_queued_messages=70000,
                  linger_ms=5 * 1000,
                  block_on_queue_full=True,
+                 max_request_size=1000012,
```

Comment:
@jofusa Any particular reason this deviates from the JVM producer's default of `1048576`?